src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -0,0 +1,285 @@
+<Project>
+  <PropertyGroup>
+    <OutputInRepoRoot>true</OutputInRepoRoot>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="..\Microsoft.AspNetCore.App.props" />
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <RuntimeIdentifier>$(TargetRuntimeIdentifier)</RuntimeIdentifier>
+    <!-- Even though RuntimeIdentifier is set, shared framework projects are not self-contained projects -->
+    <SelfContained>false</SelfContained>
+    <SharedFxName>Microsoft.AspNetCore.App</SharedFxName>
+    <PackageId>$(MSBuildProjectName).$(RuntimeIdentifier)</PackageId>
+    <IsShippingPackage>true</IsShippingPackage>
+
+    <PackageDescription>Provides a default set of APIs for building an ASP.NET Core application. Contains assets used for self-contained deployments.
+
+This package is an internal implementation of the .NET Core SDK and is not meant to be used as a normal PackageReference.
+    </PackageDescription>
+    <PackageTags>aspnetcore;shared-framework</PackageTags>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+
+    <!-- Subject to change: see https://github.com/dotnet/designs/pull/50 -->
+    <PackageType>RuntimePack</PackageType>
+    <!-- NuGet appends target framework to this value. Example: runtimes/win-x64/nativeassets/netcoreapp3.0/ -->
+    <BuildOutputTargetFolder>runtimes/$(RuntimeIdentifier)/nativeassets/</BuildOutputTargetFolder>
```

Comment:
Totally guessed on what this should be. @dsplaisted what does the SDK currently expect for runtime packs?

Reply:
Is this the path to assets within the package?  Currently the SDK looks in `runtimes/<rid>/lib/netcoreapp3.0` for managed assets and `runtimes/<rid>/native` for native assets.

Feel free to stay with that layout for now.  Alternatively, you can just put them into `data/native` and `data/managed` (which is closer to what I'd like to eventually land on) and I'll update the SDK to consume them from there.

There does need to be a way to distinguish between the native and managed assets though, as they are written differently to the deps file.

Reply:
I'll update to `runtimes/<rid>/lib/<tfm>` and `runtimes/<rid>/native` before merging.